frontend/ui/widget/progresswidget.lua

```diff
@@ -42,7 +43,8 @@ local ProgressWidget = Widget:new{
     bordersize = Screen:scaleBySize(1),
     bordercolor = Blitbuffer.COLOR_BLACK,
     bgcolor = Blitbuffer.COLOR_WHITE,
-    rectcolor = Blitbuffer.COLOR_DIM_GRAY,
+    nlcolor = Blitbuffer.COLOR_LIGHT_GRAY,
```

Comment:
I thought I had commented on that, but I can't find my comment anymore :/
So, re-mentionning it (but your call).
`nlcolor` (`nl` for non linear) and the use of `self.flows` (which should be documented with `flows = nil -- small comment` in the constructor) might have better more generic names in this generic "ProgressWidget".
"secondaries" or something else generic ?